### ISSUE TITLE
Creating HBAR accounts with Atomic Wallet

### ISSUE THREAD

**Issue by Cooper-Kunz** [hashgraph/hedera-heroes#10]
## Suggestion

A number of users are still having issues creating new HBAR wallets/accounts. 

If someone can create a new video tutorial demonstrating how to open an HBAR account w/ [Atomic Wallet](https://atomicwallet.io/), store their account details, and add it to YouTube for the community, that'd be great.

## Related

Video tutorials for other wallets would also be great. 

I'll create additional issues/suggestions for - BRD, MyHbarWallet, Ledger, and others.

**Comment by christianholman** [comment]
https://www.youtube.com/watch?v=VTQg2Ec_KeM went ahead and created this for Ivy Wallet

**Comment by whomatters** [comment]
sorry for being noob, but so, this issue is open and if i acomplish the mission of recording the video of creating a heredra wallet address in Atomic wallet (and do it good) i will earn some hbar?

**Comment by christianholman** [comment]
> sorry for being noob, but so, this issue is open and if i acomplish the mission of recording the video of creating a heredra wallet address in Atomic wallet (and do it good) i will earn some hbar?

Yep, you'll earn 100 USD worth of HBAR for contributing to spreading the word about Hedera and helping the community.

**Comment by whomatters** [comment]
Ok, got it!
Beleave me, i´m slow as a tortoise... but i will try it, on my way in 3,2,1.

**Comment by whomatters** [comment]
https://youtu.be/pu306KUv4hM
Please, don´t laught out loud, beleave it or not this is my very first video, you can take it as a "preview" (?)
I´m so exited that my english words runed away and my microphone it isn´t good. I will try to make it better, just testing another mic/devices  T_____T

**Comment by Cooper-Kunz** [comment]
Hi @whomatters - this is looking great! Is it possible to increase your microphone volume? Otherwise this LGTM! 

Feel free to send me an email to cooper@hedera.com if you need any help or have questions.

**Comment by Cooper-Kunz** [comment]
> > sorry for being noob, but so, this issue is open and if i acomplish the mission of recording the video of creating a heredra wallet address in Atomic wallet (and do it good) i will earn some hbar?
> 
> Yep, you'll earn 100 USD worth of HBAR for contributing to spreading the word about Hedera and helping the community.

Hey @christianholman - would you like to apply this video for a Heroes reward? If so, go ahead and make an issue/suggestion for it and we can queue up some social promotions (or if you want to promote it on twitter/telegram/etc we can bump)
